nodes/nomos-node/node/src/lib.rs

```diff
@@ -82,7 +82,7 @@ impl StorageSerde for Wire {
 /// Membership used by the DA Network service.
 pub type NomosDaMembership = FillFromNodeList;
 pub type DaMembershipStorage = MockStorage;
-pub type DaNetworkApiAdapter = HttApiAdapter<DaMembershipHandler<NomosDaMembership>>;
+pub type DaNetworkApiAdapter = HttApiAdapter<DaMembershipHandler<NomosDaMembership>, DaAddressbook>;
```

Comment:
I did not create `da-network` generic over Addresbooko, but have exported public type `DaAddressbook` which. is set to the mock. 
The reason was to avoid the type propagation. Let me know what you think @danielSanchezQ